CMakeLists.txt

```diff
@@ -331,7 +331,7 @@ endif ()
 
 message(STATUS "Build Type: ${CMAKE_BUILD_TYPE}")
 
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wno-strict-aliasing")
 
```

Comment:
We are breaking strict aliasing at 
reader-internal.cc:282: `uint32_t metadata_len = *reinterpret_cast<uint32_t*>(footer_buffer);`